TESTING.md

```diff
@@ -6,41 +6,68 @@ This document describes pointers and notes for testing the GPII infrastructure i
 
 This section contains a tutorial to test the frontend GPII application against the backend. Relevant calls against the backend are noted after each action.
 
-### Testing key-in/key-out and basic settings
+### Testing login/logout and basic settings
 
-1. Obtain a copy of the gpii/windows repository:
+1. Obtain a copy of the gpii/gpii-app repository:
 
 ```bash
-$ git clone git@github.com:GPII/windows.git
+$ git clone https://github.com/GPII/gpii-app.git
+
+```
+
+Note that as of 9/19/2018, the full functionality that works against the cloud only exists on the `feds-audit` branch. Because of this, you should checkout that branch instead of working on master:
+
+```
+$ git checkout feds-audit
 ```
 
-2. Use vagrant to spin up the instance.
+2. Create a new configuration in `configs/app.cloud.json`, containing the following contents:
+
+```
+{
+    "type": "gpii.appWithTaskTray",
+    "options": {
+        "gradeNames": ["fluid.component"],
+        "distributeOptions": {}
+    },
+    "mergeConfigs": [
+        "%gpii-app/configs/app.base.json",
+        "%gpii-universal/gpii/configs/gpii.config.untrusted.development"
+     ]
+}
+```
+
+3. Spin up the vagrant box:
 
 ```bash
-$ cd windows
+$ cd gpii-app
 
 $ vagrant up
 ```
 
-3. Once the windows image is spun up, open a command prompt within the virtual machine and spin up the components locally.
+4. When the machine is up and running, open a command prompt and run the application:
 
-```shell
-$ cd C:\Vagrant
+```
+$ cd c:\vagrant
 
 $ SET GPII_CLOUD_URL=http://flowmanager.<CLUSTER_DNS>
 
-$ SET NODE_ENV=gpii.config.untrusted.development
+$ ./node_modules/.bin/electron . ./configs app.cloud
```

Comment:
Inside of a "Command prompt" I have to use backslashes in the path to electron instead.

Reply:
@amatas - Thanks for the feedback. Branch has been updated with your suggestion.